videoencoder.cpp

```diff
@@ -320,37 +318,45 @@ main (int argc, char *argv[])
 	sleep(1);
 	continue;
       }
-      // better don't take half lines
-      if (line[strlen(line) - 1] == '\n') {
-	line[strlen (line) - 1] = 0;
-	cmd_offset = ftell(cmd);
-	fclose(cmd);
-      } else {
-	fclose(cmd);
-	sleep(1);
-	continue;
-      }
-      //printf ("I got '%s'\n", line);
-
-      if (line[0] == 'E')
-	{
-	  const char *filename = line + 2;
-
-	  Mat image;
-	  image = imread (filename, CV_LOAD_IMAGE_COLOR);
-
-	  if (!image.data)	// Check for invalid input
-	    {
-	      std::cout << "Could not open or find the image" << std::endl;
-	      return -1;
-	    }
-
-	  rgb_to_yuv (&image, ycbcr);
-
-	}
+        // better don't take half lines
+        if (line[strlen(line) - 1] == '\n') {
+                line[strlen (line) - 1] = 0;
+                cmd_offset = ftell(cmd);
+                fclose(cmd);
+        } else {
+                fclose(cmd);
+                sleep(1);
+                continue;
+        }
+
+        if (line[0] == 'E') {
+                const char *filename = line + 2;
```

Comment:
The indenting style is now totally messed up. I guess I will do an emacs session with it when you're done :)